test/UnitTests/MSTest.CoreAdapter.Unit.Tests/Discovery/AssemblyEnumeratorTests.cs

```diff
@@ -164,33 +151,30 @@ public void GetTypesShouldLogWarningsWhenReflectionFailsWithLoaderExceptions()
 
     #region GetLoadExceptionDetails tests
 
-    [TestMethodV1]
     public void GetLoadExceptionDetailsShouldReturnExceptionMessageIfLoaderExceptionsIsNull()
     {
-        Assert.AreEqual(
-            "DummyMessage\r\n",
+        Verify(
+            "DummyMessage\r\n" ==
```

Comment:
Keep the code as-is but for the future we should decide upon some styling preference. I tend to prefer having the operator on the beginning of the new line because I can more easily abstract content.

For example,
```csharp
if (expression1 ==
    expression2)
{
}
```

forces me to read the full condition to see we are doing some equality. While this code

```csharp
if (expression1
    == expression2)
{
}
```

reads IMO better. Obviously you need to imagine that `expression1` and `expression2` are larger content.